workloads/router-perf-v2/ingress-performance-large.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/bash -e
+set -e
+
+. common.sh
+export NUMBER_OF_ROUTES=${NUMBER_OF_ROUTES:-500}
+CLIENTS="1 20 80"
```

Comment:
Why override clients?   Can we default it to this value or accept a value set by the user?  I can see where being able to configure it would be useful - e.g. to shorten a run to validate the test is working well.